### DISCUSSION TITLE
zkSync bridge testnet fees seem super high

### DISCUSSION THREAD

**Discussion by SneakyPM** [Cyfrin/foundry-full-course-cu#2347] [General]
with the small drips from the test faucets it'll take some time to have a high enough balance for an eye watering _(if it was real money)_ ~0.044 ETH fee, is this normal?
![image](https://github.com/user-attachments/assets/b7af0fc1-8d03-457b-83e9-b2616214b7ba)

PS: the balance in the source wallet is ~0.0237, I noticed in the video of the _"making your first transaction on zkSync"_ lesson the fee is also high, but it does not stop the transaction with an _"insufficient balance"_ error like above, and now after successfully bridging _(got more funds from a faucet to cover the high fee)_ I see the actual fee shown in the block explorer is substantially lower than the validation step that prevented the transaction previously i.e. 0.0009 actual vs 0.044 estimated

**Comment by EngrPips** [comment]
Interesting. I didn't notice this issue when I completed this section, and I am unsure of the reason for this discrepancy.

**Comment by AboloreDev** [comment]
Maybe there was a lot of traffic at the time of bridging as this increases gas fees most times.

**Reply by EngrPips** [reply]
Interesting.

**Comment by Totiruzi** [comment]
I am having this same issue but I have 0.05 Sapolia in my wallet
```Error
Insufficient ETH balance on Ethereum Sepolia Testnet to cover the fee. We recommend having at least 0.004420299427175108 ETH on Ethereum Sepolia Testnet for deposit.
```
Did you finally bridge?

**Reply by EngrPips** [reply]
How much `sepolia-eth` did you try to bridge?
